src/zmqpp/loop.hpp

```diff
@@ -125,18 +127,18 @@ namespace zmqpp
             void update();
         };
 
-        typedef std::pair<zmq_pollitem_t, Callable> PollItemCallablePair;
```

Comment:
does changing these types break api backward compatibility?

Reply:
I think no. it's private type. But size of class will be changed, so recompile of clients is required.

bluca, please say me what is better:
1. register onAfterRemove callback at add() time
2. register onAfterRemove callback at remove() time

First method is implemented now. But second seems more clean for user and more simple to implement.